query/src/interpreters/stream_addon.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2021 Datafuse Labs.
```

Comment:
Better to move to common/streams?

Reply:
No, depending on planners & expression executor.